istio/README.md

```diff
@@ -197,9 +193,9 @@ instances:
 
 ```
 
-#### Openmetrics V1 configuration (Legacy) with generic metric collection
+#### OpenMetrics V1 configuration (Legacy) with generic metric collection
 
-Be sure to exclude Istio and Envoy metrics from your configuration to avoid high custom metrics billing. Use `ignore_metrics` if using the Openmetrics V1 configuration (`prometheus_url` enabled).
+Be sure to exclude Istio and Envoy metrics from your configuration to avoid high custom metrics billing. Use `ignore_metrics` if using the OpenMetrics V1 configuration (`prometheus_url` enabled).
```

Comment:
```suggestion
Be sure to exclude Istio and Envoy metrics from your configuration to avoid high custom metrics billing. Use `ignore_metrics` if `prometheus_url` is enabled.
```